app/views/question_flow/forms/_assets.html.slim

```diff
@@ -46,57 +60,98 @@
     .govuk-button-group
       button.govuk-button.govuk-button--secondary type="button" data-add-another-role="add" = t("question_flow.assets.add_bank_account")
 
-    = form.govuk_fieldset legend: { text: t("question_flow.#{i18n_key}.investments.label") } do
+    - if !FeatureFlags.enabled?(:assets_without_conditional_reveals, @check.session_data)
+      = form.govuk_radio_buttons_fieldset :investments_relevant,
+              legend: { text: t("question_flow.#{i18n_key}.investments.label") },
+              hint: { text: t(investment_hint_key) } do
+        = render "shared/radio_button_set",
+                form:,
+                field_name: :investments_relevant,
+                buttons: [{ value: true,
+                        options: { label: { text: t("generic.yes_choice") },
+                                hint: -> { content_for(:amount_hidden_description) } } },
+                        { value: false, options: { label: { text: t("generic.no_choice") } } }]
+
+          = render "shared/money_input",
+                  form:,
+                  field: :investments,
+                  width: 5,
+                  label_text: t("question_flow.#{i18n_key}.investments.input")
+
+          - if show_smod_content
```

Comment:
I don't believe the SMOD checkbox appears to be working for investments and valuables. I'm not sure if it's to do with the HTML or the form itself! Checkbox is not appearing when it should.

I went on the deployment and did a controlled check, not asylum supported, no partner and yes to passported benefits - expected to see SMOD checkbox and did not. (it did appear for bank accounts correctly)

Reply:
weird - SMOD checkbox definitely working now, there's even a test for its presence